examples/compression/bios_bit_unpack/source/main.c

```diff
@@ -67,7 +67,7 @@ void bit_unpack_1bpp(void)
         fontTilesLen, // Source size, in bytes
         1, // Source bit depth
         4, // Destination bit depth
-        1 | (1 << 31) // Add one to all converted values.
```

Comment:
Can we turn this into a constant in `<nds/bios.h>`? This will have the free benefit of making the code more readable, too.

Reply:
Yeah, I agree. If we're going to fix this small issue, we might as well fix it properly. Something like `BIOS_UNPACK_OFFSET_ADD_ALWAYS` would be fine.